Makefile

```diff
@@ -1,14 +1,18 @@
 cs:
 	php-cs-fixer fix --verbose
+	npm run lint -- --fix
```

Comment:
If i advise someone to run make cs to fix the build, and they don't have npm (and they don't care because they don't work on the JS part), things are going to get complicated. How about :

```
backend_cs:
	php-cs-fixer fix --verbose

backend_cs_dry_run:
	php-cs-fixer fix --verbose --dry-run

frontend_cs:
	npm run lint -- --fix

frontend_cs_dry_run:
	npm run lint

cs: backend_cs frontend_cs

cs_dry_run: backend_cs_dry_run frontend_cs_dry_run
```

?